### PR TITLE
OKTA-567446 Add authenticators.constraints.aaguidGroups to MFA_ENROLL in Policy API

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Add authenticators.constraints.aaguidGroups definition to the Authenticator enrollment policy object for the WebAuthn allowlist feature. Backport reference PR 247 in oas3.
- **Is this PR related to a Monolith release?** Yes, doc w/ RN in 2023.02.0 (orig in 2023.01.1)

### Preview:
https://63cb57f86de26f7e378bbbc8--reverent-murdock-829d24.netlify.app/docs/reference/api/policy/#policy-authenticator-object

### Resolves:

* [OKTA-567446](https://oktainc.atlassian.net/browse/OKTA-567446)

### CODE REVIEW

**Review by ManuMalhotra-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -1246,6 +1247,39 @@ The authenticator enrollment policy controls which authenticators are available
 | ---       | ---                                            | ---                                      | ---      | ---           |
 | self      | Requirements for the user-initiated enrollment | `NOT_ALLOWED`, `OPTIONAL`, or `REQUIRED` | Yes      | `NOT_ALLOWED` |
 
+#### Policy Authenticator Constraints object
+
+<ApiLifecycle access="ie" />
+<ApiLifecycle access="ea" />
+
+> **Note:** Allow List for FIDO2 (WebAuthn) Authenticators is an [Early Access](/docs/reference/releases-at-okta/#early-access-ea) (Self-Service) feature. You can enable the feature for your org from the **Settings** > **Features** page in the Admin Console.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:1255]:
```suggestion
> **Note:** Allow List for FIDO2 (WebAuthn) Authenticators is an [Early Access](/docs/reference/releases-at-okta/#early-access-ea) (Self-Service) feature. Enable the feature for your org from the **Settings** > **Features** page in the Admin Console.
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 2]
```diff
@@ -1246,6 +1247,39 @@ The authenticator enrollment policy controls which authenticators are available
...
+
+You can configure which FIDO2 WebAuthn authenticators are allowed in your org for new enrollments by defining WebAuthn authenticator groups, then specifying which groups are in the allow list for enrollments. The authenticators in the group are based on FIDO Alliance Metadata Service that is identified by name or the Authenticator Attestation Global Unique Identifier ([AAGUID](https://support.yubico.com/hc/en-us/articles/360016648959-YubiKey-Hardware-FIDO2-AAGUIDs)) number. These groups are defined in the [WebAuthn authenticator methods](/docs/reference/api/authenticators-admin/#example-webauthn-authenticator) endpoint.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:1257]:
```suggestion
Configure which FIDO2 WebAuthn authenticators are allowed in your org for new enrollments by defining WebAuthn authenticator groups, then specifying which groups are in the allow list for enrollments. The authenticators in the group are based on FIDO Alliance Metadata Service that's identified by name or the Authenticator Attestation Global Unique Identifier ([AAGUID](https://support.yubico.com/hc/en-us/articles/360016648959-YubiKey-Hardware-FIDO2-AAGUIDs)) number. These groups are defined in the [WebAuthn authenticator methods](/docs/reference/api/authenticators-admin/#example-webauthn-authenticator) endpoint.
```

**Review by susanharper-okta — APPROVED** [review]
two nits - approving not to hold you up

**Review by petermiller-okta — APPROVED** [review]

**Pull request merged**